src/Parser/EnhancementShaman/Modules/ShamanCore/Pandemic.js

```diff
@@ -0,0 +1,54 @@
+import SPELLS from 'common/SPELLS';
+
+import Module from 'Parser/Core/Module';
+
+const PANDEMIC_THRESHOLD = 11500;
+const debug = false;
+
+class Pandemic extends Module {
+  flametongueTimestamp = 0;
+  earlyRefresh = 0;
+  refreshTimestamps = [];
```

Comment:
If you're not planning on making something with this var, maybe put debug conditionals with the 2 `push` calls below to preserve memory (or remove it altogether if you're confident in this module being finished and correct).

Reply:
Debating whether to add the times in the log where it was refreshed early. I'll wrap debug in it for now until I decide for sure which way to go.